docs/docsite/rst/reference_appendices/faq.rst

```diff
@@ -665,7 +676,8 @@ This works for all overridden connection variables, like ``ansible_user``, ``ans
 How do I fix 'protocol error: filename does not match request' when fetching a file?
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-Newer releases of OpenSSH have a `bug <https://bugzilla.mindrot.org/show_bug.cgi?id=2966>`_ in the SCP client that can trigger this error on the Ansible controller when using SCP as the file transfer mechanism::
+Newer releases of OpenSSH have a `bug <https://bugzilla.mindrot.org/show_bug.cgi?id=2966>`_ in the SCP client
```

Comment:
added line break

Reply:
Newer is so unclear, newer than what ? time of writing ? the reader does not know when it was. time of reading ? will it still be true? 
I wish people writing docs gives clearer information, like "starting release 3.04 ..."